ui.frontend/src/view/FormContainer.js

```diff
@@ -20,6 +20,9 @@ export default class FormContainer {
         this._path = params._path;
         this._fields = {};
         this._deferredParents = {};
+        if (params._prefillData) {
+            this._model.importData(params._prefillData.data);
+        }
```

Comment:
I think better would be 
`createFormInstance({...params._formJson, data: _params._prefillData})`